app/js/components/manageApps/ManageApps.jsx

```diff
@@ -289,67 +292,85 @@ export default class ManageApps extends React.Component {
-        <div className="progress-bar" style={{width: uploadStatus + "%"}} />
+        <div className="progress-bar progress-bar-striped" style={{width: uploadStatus + "%"}} />
         <span>{uploadStatus + "%"} Complete</span>
       </div>
     );
 
     return (
       <div>
-        <div className="container-fluid">
-          {this.state.showMsg ? alert : null}
-          {showProgress === true ? progressBar : null}
+        <div className="main-home-page" id="body-wrapper">
+          <div className="row">
+            <div className="col-sm-6">
+              <h2>Add-on Manager</h2>
+            </div>
+            <div className="col-sm-6 manage-settings">
+              <div className="pull-right">
+                <Link to="/manageSettings" className="manage-settings-button">
+                  <i className="glyphicon glyphicon-cog settings-icon" id="icon-btn" />
+                </Link>
+              </div>
+            </div>
+          </div>
+          
+          <div className="home-page-body">
+            <div className="container-fluid">
+              <div id="notification-wrapper">
+                {this.state.showMsg ? alert : null}
+                {showProgress === true ? progressBar : null}
+              </div>
 
-          <h4 id="manageApps">Addon Manager</h4>
+              <AddAddon 
+                handleClear={this.handleClear}
+                handleUpload={this.handleUpload}
+                displayManageOwaButtons={this.displayManageOwaButtons}
+                displayManageOwaButtonsState={this.state.displayManageOwaButtons}
+              />
 
-          <AddAddon 
-            handleClear={this.handleClear}
-            handleUpload={this.handleUpload}
-            displayManageOwaButtons={this.displayManageOwaButtons}
-            displayManageOwaButtonsState={this.state.displayManageOwaButtons}
-          />
-          <div className="manage-app-table col-sm-12">
-            <div className="search-add-on">
-              <i className="glyphicon glyphicon-search" />
-              <input 
-                type="text" 
-                id="search-input" 
-                onKeyUp={this.searchAddOn} 
-                placeholder="Search for an add on.."/>
+              <div className="manage-app-table col-sm-12">
+                <div className="search-add-on">
+                  <i className="glyphicon glyphicon-search" />
+                  <input 
+                    type="text" 
+                    id="search-input" 
+                    onKeyUp={this.searchAddOn} 
+                    placeholder="Search for an add on.."/>
+                </div>
+                <table className="table table-bordered table-striped table-hover">
+                  <thead>
+                    <tr>
+                      <th>Logo</th>
+                      <th>Name</th>
+                      <th>Developer</th>
+                      <th>Version</th>
+                      {this.state.install == false ? 
```

Comment:
Can we use the `Identity` operator here (===) as opposed to using the `equality` operator (==)?

Reply:
That piece of code was not written by me, it was part of the changes I pulled after the other PRs were merged, but I'm sure gonna fix it.